api/db.py

```diff
@@ -85,37 +92,77 @@ def get_data_list(start_query, limit, offset, order_by, qs):
 
 # create query that returns data list
 def create_query_list(start_query, limit: int = 0, offset: int = 0, order_by=None, qs: str | None = None):
+    # options: limit, offset, order_by
     where_optional_args = []
     # options: limit, offset, order_by
     other_optional_args = []
     # values to put in placeholders in query
     values = []
-
-    used_clause = ["limit", "offset", "order_by"]
-
+    crc32c=False
```

Comment:
`crc32c` is a boolean variable used to handle cases where the input for a argument is of fomart `crc32c`: a string with a length of 6, followed by `==`. beacuse I use to split my query with `=` (`key, val = item.split('=')`), this cause some problem with getting right the other arguments in the query. This variable tells if we got `crc32c` format.
Here:
```
if item[-2:]=='==':
                item = item[:-2]
                crc32c=True
```
I check if we got crc32c format, if so we remove `==` so it will not disturb the arguments splitting of query.
Here:
```
if crc32c:
        val+='=='
        crc32c=False
```
I check if it's `crc32c` format argument,if so we adding back `==` so in the database will be found the correct value.